src/world.cpp

```diff
@@ -64,29 +68,46 @@ World::World(const std::string& biome_name, MapTile_t tile_type, size_t seed) :
         3, 3, macro_tile_size, height, seed, biome_, get_test_map(tile_type)
     ) {}
 
+void
+World::mark_for_update(ChunkIndex chunk_pos) {
+    chunks_to_update_.insert(chunk_pos);
+}
+
+void
+World::mark_for_update(TerrainDim3 tile_sop) {
+    if (!terrain_main_.in_range(tile_sop))
+        return;
+    Dim chunk_pos = terrain_main_.get_chunk_from_tile(tile_sop);
+    mark_for_update(chunk_pos);
+}
+
+// Should not be called except by lambda function
 __attribute__((optimize(2))) void
```

Comment:
don't need this anymore i guess
```suggestion
void
```

Reply:
why not? It takes so long to run.

Reply:
*something something* MSVC *something something*

Reply:
how much does this really help?